third_party/move/move-vm/runtime/src/loader/function.rs

```diff
@@ -551,7 +544,7 @@ impl Function {
     /// immutable (public), also store.
     pub fn abilities(&self) -> AbilitySet {
         let result = AbilitySet::singleton(Ability::Copy).add(Ability::Drop);
-        if !self.is_friend_or_private {
+        if self.is_public() {
```

Comment:
Should this not be `is_persistent()`?

Reply:
Good catch: indeed, persistent should be storable, changed.